src/test/java/teammates/test/cases/ui/browsertests/InstructorHomePageUiTest.java

```diff
@@ -363,40 +363,54 @@ public void testCopyToFsAction() throws Exception {
         ______TS("Submit empty course list: Home Page");
         
         homePage.clickFsCopyButton(courseId, feedbackSessionName);
-        homePage.waitForModalToLoad();
-        homePage.clickFsCopySubmitButton();
-        homePage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_COPY_NONESELECTED);
+        homePage.fsCopyModal.waitForModalToLoad();
+        homePage.fsCopyModal.clickSubmitButton();
+        homePage.fsCopyModal.waitForFormSubmissionErrorMessagePresence();
+        assertTrue(homePage.fsCopyModal.isFormSubmissionStatusMessageVisible());
+        homePage.fsCopyModal.verifyStatusMessage(Const.StatusMessages.FEEDBACK_SESSION_COPY_NONESELECTED);
+        
+        homePage.fsCopyModal.clickCloseButton();
         
         ______TS("Copying fails due to fs with same name in course selected: Home Page");
         
         homePage.clickFsCopyButton(courseId, feedbackSessionName);
-        homePage.waitForModalToLoad();
-        homePage.fillCopyToOtherCoursesForm(feedbackSessionName);
+        homePage.fsCopyModal.waitForModalToLoad();
+        homePage.fsCopyModal.fillFormWithAllCoursesSelected(feedbackSessionName);
         
-        homePage.clickFsCopySubmitButton();
+        homePage.fsCopyModal.clickSubmitButton();
         
         String error = String.format(Const.StatusMessages.FEEDBACK_SESSION_COPY_ALREADYEXISTS, feedbackSessionName, courseId);
+        homePage.fsCopyModal.waitForFormSubmissionErrorMessagePresence();
+        assertTrue(homePage.fsCopyModal.isFormSubmissionStatusMessageVisible());
+        homePage.fsCopyModal.verifyStatusMessage(error);
         
-        homePage.verifyStatus(error);
+        homePage.fsCopyModal.clickCloseButton();
         
         ______TS("Copying fails due to fs with invalid name: Home Page");
         
         homePage.clickFsCopyButton(courseId, feedbackSessionName);
-        homePage.waitForModalToLoad();
-        homePage.fillCopyToOtherCoursesForm("Invalid name | for feedback session");
+        homePage.fsCopyModal.waitForModalToLoad();
+        homePage.fsCopyModal.fillFormWithAllCoursesSelected("Invalid name | for feedback session");
         
-        homePage.clickFsCopySubmitButton();
+        homePage.fsCopyModal.clickSubmitButton();
+        homePage.fsCopyModal.waitForFormSubmissionErrorMessagePresence();
+        assertTrue(homePage.fsCopyModal.isFormSubmissionStatusMessageVisible());
         
-        homePage.verifyStatus("\"Invalid name | for feedback session\" is not acceptable to TEAMMATES as feedback session name because it contains invalid characters. All feedback session name must start with an alphanumeric character, and cannot contain any vertical bar (|) or percent sign (%).");
+        homePage.fsCopyModal
+                .verifyStatusMessage("\"Invalid name | for feedback session\" is not acceptable to TEAMMATES as feedback session " 
```

Comment:
May be more neat to break at the start of the string instead of at `.verifyStatusMessage`.
Outside the scope of this issue, but why are there no constants for these messages, anyway?